context/src/main/java/io/opentelemetry/context/ForwardingScheduledExecutorService.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.context;
+
+import java.util.concurrent.ScheduledExecutorService;
+
+/** A {@link ScheduledExecutorService} that implements methods that don't need {@link Context}. */
+abstract class ForwardingScheduledExecutorService extends ForwardingExecutorService
```

Comment:
Unused if you follow my advice: https://github.com/open-telemetry/opentelemetry-java/pull/6712/files#r1765751316